### PR TITLE
VBM-16: splash screen

### PR DESCRIPTION
**Pull request by mazurGit**

### CODE REVIEW

File: packages/mobile/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "react-hook-form": "7.33.1",
     "react-i18next": "11.18.3",
     "react-native": "0.69.3",
+    "react-native-bootsplash": "^4.3.2",
```
Comment (AleksandrTermenzhy) [inline packages/mobile/package.json:36]: --save-exact

File: packages/mobile/src/index.tsx [hunk 2]
```diff
@@ -5,12 +5,16 @@ import 'fast-text-encoding';
 import { Navigation } from '~/navigation/navigation';
 import { store } from '~/store/store';
 import { NavigationDarkTheme, NavigationTheme } from '~/config/config';
-import { useColorScheme } from '~/hooks/hooks';
+import { useColorScheme, useEffect } from '~/hooks/hooks';
+import RNBootSplash from 'react-native-bootsplash';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/index.tsx:9]: Move it to the top imports section, please

File: packages/mobile/src/index.tsx [hunk 3]
```diff
@@ -5,12 +5,16 @@ import 'fast-text-encoding';
...
 import { i18 } from './localization/localization';
 
 i18();
 
 const App: FC = () => {
+  useEffect(() => {
+    RNBootSplash.hide({ fade: true });
+  }, []);
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/index.tsx:17]: Again, this belongs to the `NavigatorContainer`'s `onReady` callback

File: packages/mobile/package.json (hunk 1 above)
Comment (mazurGit) [reply packages/mobile/package.json]: fixed

File: packages/mobile/src/index.tsx (hunk 2 above)
Comment (mazurGit) [reply packages/mobile/src/index.tsx]: fixed

File: packages/mobile/src/index.tsx (hunk 3 above)
Comment (mazurGit) [reply packages/mobile/src/index.tsx]: fixed

**Review by AleksandrTermenzhy — APPROVED** [review]

**Pull request merged**